README.md

```diff
@@ -173,7 +173,7 @@ cio.trackAnonymous(anonymous_id, {
 
 #### Anonymous invite events
 
-If you previously sent [invite events](https://customer.io/docs/anonymous-invite-emails/), you can achieve the same functionality by sending an anonymous event with an empty string for the anonymous identifier. To send anonymous invites, your event *must* include a `recipient` attribute. 
+If you previously sent [invite events](https://customer.io/docs/anonymous-invite-emails/), you can achieve the same functionality by sending an anonymous event with an empty string for the anonymous identifier. To send anonymous invites, your event _must_ include a `recipient` attribute.
```

Comment:
What was the reason for moving from `*` to `_`?

Reply:
@mike-engel Actually, it's from pretty-quick :) Didn't see it.